pkg/table-router/router.go

```diff
@@ -39,6 +89,36 @@ func (t *TableRule) Valid() error {
 		return errors.New("target schema of table route rule should not be empty")
 	}
 
+	if t.Extractors.TableExtractor != nil {
+		re, err := regexp.Compile(t.Extractors.TableExtractor.TableRegexp)
+		if err != nil {
+			return errors.New("table extractor table regexp illegal")
```

Comment:
it's better to also display the original regexp string. and same for SchemaExtractor, SourceExtractor